pkgs/tools/misc/zsjed/default.nix

```diff
@@ -0,0 +1,62 @@
+{ coreutils
+, fetchurl
+, findutils
+, gawk
+, gnused
+, lib
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zsjed";
+  version = "0.3";
+
+  src = fetchurl {
+    url = "https://ytrizja.de/distfiles/zsjed-${version}.tar.gz";
+    sha256 = "1rcrvrqjsv80ldz7cscqbrf98jb0yd2in97irrzlpask699z5k29";
+  };
+
+  nativeBuildInputs = [
+    coreutils # mkdir, install
+    gnused
+  ];
+
+  # runtime deps of zsjed-mkls
+  buildInputs = [
+    coreutils # sort
+    findutils
+    gawk
+  ];
+
+  # can't check because all tools return retcode 1 by default
```

Comment:
We could work around this. Do you have tests?

Reply:
I found a workaround. Is it ok that the check has some output, or should I forward everything to `/dev/null`?